lib/publish.js

```diff
@@ -88,6 +88,7 @@ module.exports = async (pluginConfig, context) => {
   const json = {
     /* eslint-disable camelcase */
     tag_name: gitTag,
+    name: `Bump tag: ${gitTag}`,
```

Comment:
The name could **optionally** be resolved by the plugin config so that:
- if it is defined, the given value is set
- if it is not defined, this value is assigned by default
like:
```js
name: pluginConfig.name || `Bump tag: ${gitTag}`,
```